api/github.js

```
@@ -83,14 +66,38 @@ export default async function handler(req) {
       while (page <= 3) {
         try {
-          const res = await fetchWithFallback(`https://api.github.com/users/${user}/repos?per_page=100&sort=updated&page=${page}`, { 
+          const res = await fetch(`https://api.github.com/users/${user}/repos?per_page=100&sort=updated&page=${page}`, {
             headers: ghHeaders,
             signal: AbortSignal.timeout(5000)
           });
-          if (!res.ok) {
-            if (page === 1) return new Response(JSON.stringify({ error: `GitHub ${res.status}` }), { status: 502, headers });
-            break;
+          if (res.status === 403) {
+            if (page == 1) {
```

ESLint

[error] Expected '===' and instead saw '=='.

(eqeqeq)